science/projects/zahrasadat_sajjadi/01-main-content/zarebin_p1.ipynb

```diff
@@ -0,0 +1,2717 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        for i in tqdm(range(len(news_df['html']))):
</pre><p>When we use the pandas data frame, using .iterrows() is easier and&nbsp;readable than writing a for loop on range(len(df)).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>